docs/local-deploy.md

```diff
@@ -0,0 +1,58 @@
+
+# How to deploy your foam locally (using jekyll)
+
+Recipe for building your GitHub Pages site locally to preview and test changes to your site.
+
+! **Disclaimer**: This Recipe is more suited for "tech savy" foam users since it requires the usage of the command line.
```

Comment:
Really being pedantic here so I do apologise!! 😆 Correct spelling is "savvy"

Reply:
no worries ;D, thank you for spotting!